app/views/courses/_apply.html.erb

```diff
@@ -6,35 +6,9 @@
   <h2 class="govuk-heading-l" id="section-apply">Apply</h2>
 
   <% if course.has_vacancies? %>
-    <% if course.provider.supporting_dfe_apply? %>
-      <p class="govuk-body">
-        <%= link_to "Apply for this course", "https://www.apply-for-teacher-training.education.gov.uk/candidate/apply?providerCode=#{course.provider.provider_code}&courseCode=#{course.course_code}", class: "govuk-button govuk-button--start" %>
-      </p>
-    <% else %>
-      <p class="govuk-body">
-        <%= link_to 'Apply on the UCAS website', 'https://2020.teachertraining.apply.ucas.com/apply/student/login.do', class: 'govuk-link' %>. You’ll need to register with UCAS before you can apply.
-      </p>
-      <p class="govuk-body">
-        Visit Get into Teaching for <%= link_to 'guidance on applying for teacher training', 'https://getintoteaching.education.gov.uk/how-to-apply/apply', class: 'govuk-link' %>.
-      </p>
-
-      <p class="govuk-body">
-        When you apply you’ll need these codes for the Choices section of your application form:
-      </p>
-      <div class="govuk-inset-text">
-        <ul class="govuk-list">
-          <li class="govuk-!-margin-bottom-2">
-            Training provider code:
-            <span class="govuk-!-display-block govuk-!-font-size-36 govuk-!-font-weight-bold"><%= course.provider.provider_code %></span>
-          </li>
-          <li>
-            Training programme code:
-            <span class="govuk-!-display-block govuk-!-font-size-36 govuk-!-font-weight-bold"><%= course.course_code %></span>
-          </li>
-        </ul>
-      </div>
-
-    <% end %>
+    <p class="govuk-body">
+      <%= link_to "Apply for this course", "https://www.apply-for-teacher-training.education.gov.uk/candidate/apply?providerCode=#{course.provider.provider_code}&courseCode=#{course.course_code}", class: "govuk-button govuk-button--start" %>
```

Comment:
where the test for the most important bit....

Reply:
It's code that is already in the codebase. I would assume it's already tested.

Reply:
Then it would of broken the tests....

Reply:
should apply url always point to prod?

Reply:
As to whether it should always point to production. I don't see why it wouldn't. Definitely open to a reason it shouldn't, but none spring to mind.